### DISCUSSION TITLE
Implement mpl-token-metadata

### DISCUSSION THREAD

**Discussion by timaiv** [bmresearch/Solnet#394] [Ideas]
Implement the mpl-token-metadata.

Docs:

https://docs.rs/mpl-token-metadata/1.2.10/mpl_token_metadata/instruction/index.html

**Comment by hoakbuilds** [comment]
This is something that me and @tiago18c have looked into, there was the Solnet.Metaplex library being developed by someone from the community but afaik it never reached code quality in order to be published to NuGet, someone did fork it and publish it under the Solana.Metaplex namespace, but we still feel like we can probably work out something here.
